resources/views/author.blade.php

```diff
@@ -0,0 +1,69 @@
+<style>
+     .book-card {
```

Comment:
don't inline your css.. keep it externally. 
If you don't want to use bootstrap css as a library, pico css or tailwind can give you a very basic css for your project. 

Take a look https://picocss.com/